src/uct/cuda/gdr_copy/gdr_copy_iface.c

```diff
@@ -39,6 +37,15 @@ static ucs_config_field_t uct_gdr_copy_iface_config_table[] = {
          ucs_offsetof(uct_gdr_copy_iface_config_t, put_bw.shared)},
         {NULL})},
 
+    {"LAT", "get:1.4e-6,put:0.4e-6",
```

Comment:
maybe rename to LATENCY?

Reply:
I like your idea but I made it to be consistent with existent env variables (see `UCX_ROCM_COPY_LAT` and `UCX_ROCM_IPC_LAT`)

Reply:
@rakhmets @brminich WDYT? what is more consistent with existing env vars?

Reply:
we already have `LAT`
ucx_info -c | grep LAT
UCX_DISTANCE_LAT=phb:0.30us,node:0.30us,sys:0.50us
 so, imo `LAT` is ok

Reply:
ok